docs/proposals/release-notes.md

```diff
@@ -115,14 +115,15 @@ hundreds of entries.  The goal is to highlight the major changes for a release.
 
 The munger/bot option fits most cleanly into the existing workflow.
 
-The design will include:
+All accounting is managed on the master branch PR only.
```

Comment:
When we first decided this was the best approach, @eparis graciously jumped in and did the original work to bot-manage the labels very quickly.  
Unfortunately this process has been quite fluid and changes have been coming fast and furious.  This doc needs to be the 'final' process - at this stage anyway.  If anything is unclear please do comment here so we can get it into a state that we can modify the bots to handle.